src/geo/leaflet/leaflet-cartodb-webgl-layer-group-view.js

```diff
@@ -0,0 +1,49 @@
+var TC = require('tangram.cartodb');
+var LeafletLayerView = require('./leaflet-layer-view');
+
+var LeafletCartoDBVectorLayerGroupView = window.L.TileLayer.extend({
```

Comment:
As we've talked behind the scene, we should use a L.Layer, I'll change all of this :)